### PR TITLE
Upgrade golangci-lint to v1.21.0

### PR DESCRIPTION
**Pull request by titusjaka**
It must resolve memory issues in Travis CI

https://github.com/golangci/golangci-lint/releases/tag/v1.20.0

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -4,7 +4,7 @@ services:
   - redis-server
 language: go
 go:
-  - 1.13.1
+  - 1.13.x
```
Comment (litleleprikon) [inline .travis.yml]: I would not like to add this change as I want to upgrade go versions manually and keep CI builds reproducible. And this version can change separately from Dockerfiles.

**Comment by titusjaka** [comment]
> Linting is failing in travis with OOM. Seems that new version of GolancCI-Lint did not fix this. I Assume that we need to set GOGC variable again, but probably with higher value.

I set it to 50% and now it works well.

**Comment by beevee** [comment]
Each next version is getting ~10s slower :-)

**Comment by beevee** [comment]
Looks like this timing varies from build to build. Sometimes it's 160 seconds, sometimes 175. I'm going to merge this for the sake of more correct way of downloading golangci-lint, and removing deprecated config param.

**Pull request merged**